recipe/meta.yaml

```
@@ -1,41 +1,57 @@
+{% set name = "types-jsonschema" %}
 {% set version = "4.19.0.4" %}
 
 package:
   name: types-jsonschema
   version: {{ version }}
 
+
 source:
-  - url: https://pypi.io/packages/source/t/types-jsonschema/types-jsonschema-{{ version }}.tar.gz
+  - url: https://pypi.io/packages/source/t/{{ name }}/{{ name }}-{{ version }}.tar.gz
     sha256: 994feb6632818259c4b5dbd733867824cb475029a6abc2c2b5201a2268b6e7d2
-  - url: https://raw.githubusercontent.com/python/typeshed/main/LICENSE
-    sha256: 295f8538c94ae5c3043301cf7cff1c852dab6a786a8ddee471e061b40d5ecabe
+
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<36]
```

```suggestion
  skip: true  # [py<38]
```
See https://inspector.pypi.io/project/types-jsonschema/4.19.0.4/packages/92/f0/40eee69d627ced6547ca68942b50f333638250f414e6f76faffd2a2dff04/types-jsonschema-4.19.0.4.tar.gz/types-jsonschema-4.19.0.4/setup.py#line.49